js-libs/renderer/manager/LoadByStageStrategy.js

```diff
@@ -12,7 +12,7 @@ LoadByStageStrategy = Class.extend({
         //console.info('createjs.CordovaAudioPlugin.isSupported()', createjs.CordovaAudioPlugin.isSupported());
         var instance = this;
         var regex = new RegExp("^(http|https)://", "i");
-        if(regex.test(basePath)){
+        if(regex.test(basePath) && isbrowserpreview){
```

Comment:
Are we used isbrowserpreview object in renderer.min.js file anywhere? Ideally this should not depend of any global level variables. 

@pallakartheekreddy

Reply:
We are already using `isbrowserpreview` inside LoadByStageStrategy.js
.
https://github.com/project-sunbird/sunbird-content-player/blob/9fc06f842ff2bc0bd1b1143d288caed1075ada83/js-libs/renderer/manager/LoadByStageStrategy.js#L30
.
If we should use any global level variable what check we should use to ensure it is mobile play